apps/src/templates/progress/ProgressDetailToggle.jsx

```diff
@@ -49,8 +49,15 @@ const styles = {
  * A toggle that provides a way to switch between detail and summary views of
  * our course progress.
  */
-const ProgressDetailToggle = React.createClass({
-  propTypes: {
+export default connect(
+  state => ({
+    isPlc: !!state.progress.professionalLearningCourse,
+    isSummaryView: state.progress.isSummaryView,
+    hasGroups: hasGroups(state.progress)
+  }),
+  {setIsSummaryView}
+)(class ProgressDetailToggle extends React.Component {
+  static propTypes = {
```

Comment:
I would rather have the connect separate from the class TBH (i.e. at the end of the file). The syntax here seems kind of difficult to read. The pattern of having them separate also makes it easy to export a non-connected version as well as a connected version.